examples/example_cori_haswell.cfg

```diff
@@ -0,0 +1,128 @@
+# Do not edit any example cfg except for `example_generic.cfg`!
+
+[default]
+case = 20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis
+environment_commands = "source /global/common/software/e3sm/anaconda_envs/load_latest_e3sm_unified_cori-haswell.sh"
+input = /global/cscratch1/sd/forsyth/zppy_complete_run_nersc_output/20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis
+input_subdir = archive/atm/hist
+mapping_file = /global/homes/z/zender/data/maps/map_ne30pg2_to_cmip6_180x360_aave.20200201.nc
+output = /global/cscratch1/sd/forsyth/zppy_complete_run_nersc_output/20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis
+partition = haswell
+walltime = "02:00:00"
+www = /global/cfs/cdirs/e3sm/www/forsyth/zppy_complete_run_nersc_output
+
+[climo]
+active = True
+years = "51:55:2", "51:55:4",
+
+  [[ atm_monthly_180x360_aave ]]
+  frequency = "monthly"
+
+  [[ atm_monthly_diurnal_8xdaily_180x360_aave ]]
+  frequency = "diurnal_8xdaily"
+  input_files = "eam.h4"
+  vars = "PRECT"
+
+[ts]
+active = True
+frequency = "monthly"
+years = "51:55:2",
+
+  [[ atm_monthly_180x360_aave ]]
+  input_files = "eam.h0"
+
+  [[ atm_daily_180x360_aave ]]
+  frequency = "daily"
+  input_files = "eam.h1"
+  vars = "PRECT"
+
+  [[ atm_monthly_glb ]]
+  input_files = "eam.h0"
+  input_subdir = "archive/atm/hist"
+  mapping_file = "glb"
+  years = "51:61:5",
+
+  [[ land_monthly ]]
+  input_files = "elm.h0"
+  input_subdir = "archive/lnd/hist"
+  vars = "FSH,LAISHA,LAISUN,RH2M"
+
+  [[ rof_monthly ]]
+  extra_vars = 'areatotal2'
+  input_files = "mosart.h0"
+  input_subdir = "archive/rof/hist"
+  mapping_file = ""
+  vars = "RIVER_DISCHARGE_OVER_LAND_LIQ"
+
+[tc_analysis]
+active = True
+scratch = /global/cscratch1/sd/forsyth
+years = "51:53:2",
+
+[e3sm_diags]
+active = True
+grid = '180x360_aave'
+obs_ts = /global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/time-series
+ref_final_yr = 2014
+ref_start_yr = 1985
+reference_data_path = /global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/climatology
+short_name = '20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis'
+ts_num_years = 2
+years = "51:55:2", "51:55:4",
+
+  [[ atm_monthly_180x360_aave ]]
+  climo_diurnal_frequency = "diurnal_8xdaily"
+  climo_diurnal_subsection = "atm_monthly_diurnal_8xdaily_180x360_aave"
+  dc_obs_climo = "/compyfs/e3sm_diags_data/obs_for_e3sm_diags/climatology"
+  sets = "lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","enso_diags","qbo","diurnal_cycle","annual_cycle_zonal_mean","streamflow", "zonal_mean_2d_stratosphere",
+  streamflow_obs_ts = /global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/time-series
+
+  [[ atm_monthly_180x360_aave_tc_analysis ]]
```

Comment:
`e3sm_diags_atm_monthly_180x360_aave_tc_analysis_model_vs_obs_0051-0052` says "OK" for status but it doesn't produce a viewer. There is a `FileNotFoundError: [Errno 2] No such file or directory: b'/global/cfs/cdirs/e3sm/e3sm_diags/obs_for_e3sm_diags/tc_analysis/IBTrACS.NA.v04r00.nc'`.